src/legacy/Routes.tsx

```diff
@@ -499,7 +500,8 @@ class RouteApp extends React.Component<any, any> {
         <AboutModal />
         <SquooshToolModal />
         <GlitchImageModal />
-        {info?.version && <Header />}
+        {/* info?.version && <Header /> */}
+        <Header />
```

Comment:
What was fixed by this change? Code in legacy is unpredictable, so I would recommend to not change it without necessity

Reply:
When user passed login/register steps and redirected to dashboard or another page (legacy routing), header is hidden without this fix.